op-e2e/config/init.go

```diff
@@ -287,17 +287,15 @@ func initAllocType(root string, allocType AllocType) {
 				}
 			}
 
-			if allocType == AllocTypeEspressoWithoutEnclave {
-				batcherPk, err := crypto.HexToECDSA(ESPRESSO_PRE_APPROVED_BATCHER_PRIVATE_KEY)
+			// Configure Espresso allocation types
+			if allocType == AllocTypeEspresso || allocType == AllocTypeEspressoWithoutEnclave || allocType == AllocTypeEspressoWithEnclave {
```

Comment:
Why do we have `AllocTypeEspresso` when we also have `*WithEnclave` and `*WithoutEnclave`?